### PR TITLE
fix(demo): FRMW-812 rxDatePicker use local time zone

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-812

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM
- [x] QE LGTM

---
Updated the demo and midway tests to use local time zone versus the current default UTC.

### CODE REVIEW

File: src/molecules/rxDatePicker/docs/examples/rxDatePicker.simple.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 angular.module('demoApp')
 .controller('rxDatePickerSimpleCtrl', function ($scope) {
-    $scope.dateModel = new Date().toISOString().split('T')[0];
+    $scope.dateModel = moment(new Date()).format('YYYY-MM-DD', 'LL');//use local time zone
```
Comment (CITguy) [inline src/molecules/rxDatePicker/docs/examples/rxDatePicker.simple.js]: What's the `LL` parameter for?

Comment (100stacks) [inline src/molecules/rxDatePicker/docs/examples/rxDatePicker.simple.js]: It's used for setting the `moment` object to the localized date.

Comment (CITguy) [inline src/molecules/rxDatePicker/docs/examples/rxDatePicker.simple.js]: The `LL` format would generate `April 22 2016`. The model value should be in the format of `YYYY-MM-DD`. The directive will care of converting the model value to the appropriate display format (`MMMM DD, YYYY`).  

Please remove the `LL` argument. I do not believe it is necessary.

File: src/molecules/rxDatePicker/docs/rxDatePicker.midway.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('rxDatePicker', function () {
         var isoFormat = 'YYYY-MM-DD';
         var formatMonth = 'MMM';
         var formatYear = 'YYYY';
-        var today = moment(new Date());
+        var today = moment(new Date()).format(isoFormat, 'LL');//use local time zone
```
Comment (CITguy) [inline src/molecules/rxDatePicker/docs/rxDatePicker.midway.js]: Please remove the `LL` argument.

**Comment by 100stacks** [comment]
@CITguy, ok I removed `LL` argument usage from the update.

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
FYI: I'm going to trigger Travis after 6PM to verify.

**Comment by 100stacks** [comment]
@CITguy, when I tested the patch locally, I switched between different times of day and time zones to try to replicate the 6PM to Midnight UTC differential.  I think that's why I originally used the `LL` argument to explicitly use local time over UTC. ` ¯\_(ツ)_/¯ `

**Comment by CITguy** [comment]
Looks like Travis agrees. 👍

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**